pkg/cloud-provider/cloudapi/azure/azure_security_test.go

```diff
@@ -0,0 +1,251 @@
+// Copyright 2022 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package azure
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	network "github.com/Azure/azure-sdk-for-go/services/network/mgmt/2021-03-01/network"
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	"antrea.io/nephe/apis/crd/v1alpha1"
+	cloudcommon "antrea.io/nephe/pkg/cloud-provider/cloudapi/common"
+	"antrea.io/nephe/pkg/cloud-provider/securitygroup"
+)
+
+var _ = Describe("Azure Cloud Security", func() {
+	var (
+		testAccountNamespacedName = &types.NamespacedName{Namespace: "namespace01", Name: "account01"}
+		testSubID                 = "SubID"
+		credentials               = "credentials"
+		testClientID              = "ClientID"
+		testClientKey             = "ClientKey"
+		testTenantID              = "TenantID"
+		testRegion                = "eastus"
+		testRG                    = "testRG"
+		nsgID                     = "nsgID"
+		asgID                     = "asgID"
+
+		testVnet01   = "testVnet01"
+		testVnetID01 = fmt.Sprintf("/subscriptions/%v/resourceGroups/%v/providers/Microsoft.Network/virtualNetworks/%v",
+			testSubID, testRG, testVnet01)
+		testVnet02   = "testVnet02"
+		testVnetID02 = fmt.Sprintf("/subscriptions/%v/resourceGroups/%v/providers/Microsoft.Network/virtualNetworks/%v",
+			testSubID, testRG, testVnet02)
+	)
+
+	Context("SecurityGroup", func() {
+		var (
+			c          *azureCloud
+			account    *v1alpha1.CloudProviderAccount
+			selector   *v1alpha1.CloudEntitySelector
+			secret     *corev1.Secret
+			fakeClient client.WithWatch
+
+			mockCtrl                        *gomock.Controller
+			mockAzureServiceHelper          *MockazureServicesHelper
+			mockazureNwIntfWrapper          *MockazureNwIntfWrapper
+			mockazureNsgWrapper             *MockazureNsgWrapper
+			mockazureAsgWrapper             *MockazureAsgWrapper
+			mockazureVirtualNetworksWrapper *MockazureVirtualNetworksWrapper
+			mockazureResourceGraph          *MockazureResourceGraphWrapper
+			mockazureService                *MockazureServiceClientCreateInterface
+		)
+		_ = account
```

Comment:
This shouldn't be needed, can you remove it? Also in azure_test.go there is a similar instance at line 76, can you remove that too? These should be the left over thing we forgot to remove in the K8s secret PR.